test/e2e/set/scyllacluster/verify.go

```diff
@@ -116,17 +116,28 @@ func verifyScyllaCluster(ctx context.Context, kubeClient kubernetes.Interface, s
 
 	verifyPersistentVolumeClaims(ctx, kubeClient.CoreV1(), sc)
 
-	// TODO: Use scylla client to check at least "UN"
 	scyllaClient, hosts, err := utils.GetScyllaClient(ctx, kubeClient.CoreV1(), sc)
 	o.Expect(err).NotTo(o.HaveOccurred())
 	defer scyllaClient.Close()
 
 	o.Expect(hosts).To(o.HaveLen(memberCount))
 
-	status, err := scyllaClient.Status(ctx, "")
-	o.Expect(err).NotTo(o.HaveOccurred())
-	o.Expect(status.DownHosts()).To(o.BeEmpty())
-	o.Expect(status.LiveHosts()).To(o.HaveLen(memberCount))
+	allNodesUN := true
+	for _, h := range hosts {
+		statuses, err := scyllaClient.Status(ctx, h)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.Infof("Host statuses from %s: %v", h, statuses)
+		for _, s := range statuses {
```

Comment:
it should fail if node isn't found in this list.

Reply:
done